assets/src/routers/minio_router.py

```diff
@@ -155,13 +155,15 @@ async def create_bucket(
             HTTPException status 400
 
     """
+    if settings.bucket_prefix is not None:
+        bucket.name = f"{settings.bucket_prefix}-{bucket.name}"
```

Comment:
Is it only one place where `bucket.name` is used?

Reply:
No, it's not.
I've added changes to other places, where tenant name from jwt is used as bucket.name.